<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CESuperCellBuilder</class>
 <widget class="QDockWidget" name="CESuperCellBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super Cell Parameters</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="geometryComboBox">
      <item>
       <property name="text">
        <string comment="cube / block">Box</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string comment="nanoparticle">Sphere</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="aLabel">
           <property name="text">
            <string>A cut-off:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="bLabel">
           <property name="text">
            <string>B cut-off:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="cLabel">
           <property name="text">
            <string>C cut-off:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="aWidthUnits">
           <item>
            <property name="text">
             <string>length</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cell repeats</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="bWidthUnits">
           <item>
            <property name="text">
             <string>length</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cell repeats</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QComboBox" name="cWidthUnits">
           <item>
            <property name="text">
             <string>length</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cell repeats</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="cSpinBox">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="aSpinBox">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="bSpinBox">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buildButton">
        <property name="text">
         <string>Build</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
